posthog/celery.py

```diff
@@ -411,8 +419,16 @@ def graphile_worker_queue_size():
         """
         )
 
-        for (task_identifier, count) in cursor.fetchall():
-            statsd.gauge("graphile_waiting_jobs", count, tags={"task_identifier": task_identifier})
+        with pushed_metrics_registry("celery_graphile_worker_queue_size") as registry:
+            waiting_jobs_gauge = Gauge(
+                "posthog_celery_graphile_waiting_jobs",
```

Comment:
What's the reason for the gauge metric names sometimes matching the statsd one, but sometimes not?

Reply:
I think that the `posthog_celery_` prefix that most metrics already had can be useful (as they rely on celery health to be up to date), so I aligned all of them to have the prefix. Might as well do it now that we'll have to rewrite the queries anyway.